packages/flutter/lib/src/widgets/image.dart

```diff
@@ -1061,9 +1063,10 @@ class _ImageState extends State<Image> with WidgetsBindingObserver {
     _resolveImage();
 
     if (TickerMode.of(context)) {
-      _listenToStream();
+      _listenToStream(obscured: _obscured);
+      _obscured = false;
     } else {
-      _stopListeningToStream(keepStreamAlive: true);
```

Comment:
This line here _should_ have kept the frame alive. Are you able to figure out why it isn't?

Reply:
Here we are only increasing the count of `_keepAliveHandles`, which cannot prevent images from being evicted by `ImageCache`.

Reply:
https://github.com/flutter/flutter/blob/9a68bffb7b3c8d85da3a4895bba7fc5e323444fd/packages/flutter/lib/src/widgets/image.dart#L1066

Because of the extra `keepalive`, the memory of the images evicted by ImageCache was not immediately released. I filed a new issue to track it (https://github.com/flutter/flutter/issues/125608).